amy/fiscal/forms.py

```diff
@@ -92,6 +93,14 @@ def save(self, *args, **kwargs):
 
 class MembershipForm(forms.ModelForm):
     helper = BootstrapHelper(add_cancel_button=False)
+    extensions = FlexibleSplitArrayField(
+        forms.IntegerField(),
```

Comment:
I'm not as familiar with this. Should the base field be specified here?

Reply:
Yes, this is composite field. It's postgres-specific, you can read more here: https://docs.djangoproject.com/en/3.2/ref/contrib/postgres/forms/#splitarrayfield